src/cmd/cmd_sym.ml

```diff
@@ -99,30 +95,26 @@ let print_bug ~model_format ~labels ~model_out_file ~id ~no_value
       in
       Ok (Fpath.add_ext model_ext base)
     in
-    Bos.OS.File.write path (to_string model labels)
+    Bos.OS.File.write path (to_string model labels breadcrumbs scoped_symbols)
   in
-  function
-  | `ETrap (tr, model, _, _) -> (
-    Logs.err (fun m -> m "Trap: %s" (Result.err_to_string tr));
+  let output model labels breadcrumbs scoped_symbols =
     match model_out_file with
-    | Some path -> to_file path model
+    | Some path -> to_file path model labels breadcrumbs scoped_symbols
     | None -> begin
-      Logs.app (fun m -> m "%s" (to_string model labels));
+      Logs.app (fun m ->
+        m "%s" (to_string model labels breadcrumbs scoped_symbols) );
```

Comment:
It's not really ideal that we're using `to_string` here actually. We should instead define a `pp` function that works on `Format.formatter` and use it directly rather than allocating a string.